packages/lesswrong/lib/sql/Type.ts

```diff
@@ -19,9 +19,16 @@ const forceNonResolverFields = [
   "postPageDescription",
 ];
 
-export const isResolverOnly =
-  <N extends CollectionNameString>(fieldName: string, schema: CollectionFieldSpecification<N>) =>
-    schema.resolveAs && !schema.resolveAs.addOriginalField && forceNonResolverFields.indexOf(fieldName) < 0;
+export const isResolverOnly = <N extends CollectionNameString>(
+  collection: CollectionBase<N>,
+  fieldName: string,
+  schema: CollectionFieldSpecification<N>,
+) => {
+  if (collection.collectionName === "Posts" && fieldName === "moderationGuidelines") {
```

Comment:
This adds a horrible special case because there are other collections with a `moderationGuidlines` field that are still denormalized. This should only be temporary until we normalize the others as well.